references/cicd-sharedflow-pipeline/test/integration/test/integration/apikey.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Get Currency Rates Feature
```

Comment:
This isn't a currency api but isnt instead a 'shared flow test' or something

Reply:
Yes - copy paste error (embarrassing)
Fixed